src/Styles/MenuBar.less

```diff
@@ -3,17 +3,24 @@
     right: 25px;
     top: 20px;
     height: 30px;
+    border: 1px solid rgba(255,255,255,0.1);
```

Comment:
Are you sure you want `rgba(255,255,255,0.1)` rather than `rgba(25,25,25,1.0)`?  On a black background they're the same, but I think the latter will look better when the borders are over a multicolored background like the Earth.